src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java

```diff
@@ -43,7 +46,12 @@ public static TestStreamingClient create(final String baseUrl, final String subs
     public void run() {
         try {
             final String url = format("{0}/subscriptions/{1}/events?{2}", baseUrl, subscriptionId, params);
-            inputStream = new URL(url).openStream();
+            final HttpURLConnection conn = (HttpURLConnection) new URL(url).openConnection();
+            if (conn.getResponseCode() != HttpURLConnection.HTTP_OK) {
+                throw new IOException("Response code is " + conn.getResponseCode());
+            }
+            sessionId = conn.getHeaderField("X-Nakadi-SessionId");
```

Comment:
I see, we changed this field only in the API definition but not in the implementation.